pkg/kubelet/nodeshutdown/nodeshutdown_manager_linux.go

```diff
@@ -143,6 +153,24 @@ func (m *managerImpl) Admit(attrs *lifecycle.PodAdmitAttributes) lifecycle.PodAd
 	return lifecycle.PodAdmitResult{Admit: true}
 }
 
+// setMetrics sets the metrics for the node shutdown manager.
+func (m *managerImpl) setMetrics() {
+	if m.enableMetrics && m.storage != nil {
+		sta := state{}
+		err := m.storage.Load(&sta)
+		if err != nil {
+			klog.ErrorS(err, "Failed to load graceful shutdown state")
```

Comment:
nit: This should be a warning

Reply:
Structured logs have only Error and Info. Is it to modify the message content to change it from Failed to Warning?